### DISCUSSION TITLE
[Unstable] You get shot by your own gun projectile

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15193] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

bullets fired by yourself can damage you now in multiplayer

### Reproduction steps

1. start a multiplayer
2. shoot at your legs

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.0.1 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, fix coming up in the next unstable build.
